src/global.css

```diff
@@ -8,22 +8,32 @@
 
 body {
   margin: 0;
-  width: 100vh;
 }
 
 :root {
   /* colors */
   --primary-background-color: #e5e5e5;
   --primary-text-color: black;
   --primary-font: "Roboto", sans-serif;
-  --color-tabs: #a0aff4;
-  --color-tab-bucketlist: #d2c099;
-  --color-button-covid: #d98d89;
+  --color-tabs: rgb(123 104 238 / 50%);
+  --color-tab-bucketlist: rgb(255 218 185 / 50%);
+  --color-button-covid: rgb(250 128 114 / 50%);
 
   /* fonts */
   --font-text: "Roboto", sans-serif;
-  --font-size-primary: 48px;
-  --font-size-secondary: 25px;
+  --font-size-primary: 60px;
```

Comment:
I'm not sure about this, but since you are using em and rem in other places, shouldn't you do so here as well?